client/src/components/NavBarFooter.js

```diff
@@ -29,56 +29,51 @@ const Wrapper = styled.div`
   align-items: center;
 `;
 
-const NavButton = styled.button`
-  width: 40px;
-  height: 40px;
-  background-color: ${light.colors.basic};
-  outline: none;
+const NavButton = styled(Link)`
+  width: 25px;
+  height: 25px;
+  background-color: ${props => props.theme.colors.basic};
   border: none;
 `;
 
 const Highlight = styled.div`
   width: 30px;
   height: 2px;
-  background-color: ${light.colors.fontprimary};
-  position: absolute;
-  top: 0;
-`;
-
-const Highlightinactive = styled.div`
-  width: 30px;
-  height: 2px;
-  background-color: ${light.colors.basic};
+  background-color: ${props =>
+    props.active ? props.theme.colors.fontprimary : props.theme.colors.basic};
   position: absolute;
   top: 0;
 `;
 
 export default function NavBarFooter() {
+  const location = useLocation();
   return (
     <Footer>
       <Wrapper>
-        <Highlightinactive></Highlightinactive>
-        <NavButton>
-          <HomeIcon />
+        <Highlight active={location.pathname === '/'}></Highlight>
```

Comment:
If you don't have a footer, you can use short form: ` <Highlight active={location.pathname === '/'} />`